src/hazelweb/Hazel.re

```diff
@@ -8,23 +8,35 @@
 module Js = Js_of_ocaml.Js;
 module Dom = Js_of_ocaml.Dom;
 module Dom_html = Js_of_ocaml.Dom_html;
+module ResizeObserver = Js_of_ocaml.ResizeObserver;
 open Incr_dom;
 
 module Model = Model;
 module Action = Update.Action;
 module State = State;
 
 let on_startup = (~schedule_action, _) => {
-  let update_font_metrics = () => {
-    let rect =
-      JSUtil.force_get_elem_by_id("font-specimen")##getBoundingClientRect;
-    schedule_action(
-      Update.Action.UpdateFontMetrics({
-        row_height: rect##.bottom -. rect##.top,
-        col_width: rect##.right -. rect##.left,
-      }),
+  // Necessary to correctly recalculate font metrics if delay in loading font or resize of window
```

Comment:
I think this is the right way to do things so no need to explain why we don't do the wrong way.